### ISSUE TITLE
Using STL files as Custom Parts

### ISSUE THREAD

**Issue by Aiyush-G** [leozide/leocad#527]
Hi, 

I am on MacOS and would like to use .stl files that I have created in Fusion360 in LeoCad. Is this possible, if so how?

Thanks

**Comment by trevorsandy** [comment]
For this conversion, you will need to download stl2dat from http://marc.klein.free.fr/lego/stl2dat/stl2dat.html.

Cheers,

**Comment by Aiyush-G** [comment]
Thanks for the suggestion, I tried to do that, however, I am on MacOS so it couldn't run the .exe I also tried using Crossover and that didn't work. So I did a little more research to find this: https://github.com/HazenBabcock/stl-to-dat I found that it did and didn't work for some things. After reading the documentation of what you recommended above I found that It doesn't work with Binary stl files but only with ASCII stl files so I downloaded https://github.com/cmpolis/convertSTL and converted my files, it then worked. I proved this by adding it to the library.bin parts folder and ran `make` and it appeared in the application.
